### PR TITLE
Login

### LINKED ISSUE #17: Create login page for user
- login through amazon cognito 
-

### PR DESCRIPTION
**Pull request by btheard3**
resolve #17 
first attempt at login page no routes or anything is connected need someone or two or three to view and review and give me feedback.

### CODE REVIEW

File: app/src/views/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import {View, Text, StyleSheet, TextInput, Button} from 'react-native'
+
+import { Main } from './navigation'
+
+export default class Login extends React.component {
```
Comment (amberjones) [inline app/src/views/Login.jsx]: I'm not sure what this should look like integrated with the Authentication. I'd get with Titus and see whats up

Comment (btheard3) [reply app/src/views/Login.jsx]: gotcha

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```
Comment (BunnyDunker) [inline package-lock.json]: Is this file necessary?

Comment (btheard3) [reply package-lock.json]: no I can def remove it
